source/hassio/installation.markdown

```diff
@@ -61,13 +61,20 @@ This is the list of packages you need to have available on your system that will
  - network-manager
  - socat
  - software-properties-common
-
+ - docker*
+ 
+*As apt-get might install an old docker version, run the following command as root to install docker:
```

Comment:
This is not `apt-get` issue but a Debian/Ubuntu problem in general but may affect other platforms as well.